feddlib/problems/specific/Stokes_def.hpp

```diff
@@ -30,6 +30,22 @@ Problem<SC,LO,GO,NO>(parameterList, domainVelocity->getComm())
 
     this->addVariable( domainVelocity , FETypeVelocity , "u" , domainVelocity->getDimension());
     this->addVariable( domainPressure , FETypePressure , "p" , 1);
+    if(this->parameterList_->sublist("Parameter").get("Use Pressure Correction",true) && !this->getFEType(0).compare("P2") && !this->parameterList_->sublist("General").get("Preconditioner Method","Monolithic").compare("Monolithic")){ // We only correct pressure in P2 Case
```

Comment:
Hier die Erinnerung das wir einmal eine parametrListProblem.xmlerstellen sollten wo ALLE keywords drin sind mit ekurzen erklärunngen wie sie genutzt werdne un dwo - wir haben schon echt viele keywords die sonst irgendwann in vergessenheit geraten

Reply:
Ist erstmal auskommentiert. Ich lasse es mal drin, falls es nochmal gebraucht wird.